src/components/Checkbox/Checkbox.js

```diff
@@ -2,21 +2,24 @@
 Copyright (C) 2018 The Trustees of Indiana University
 SPDX-License-Identifier: BSD-3-Clause
 */
-import * as React from 'react';
+import React from 'react';
+import PropTypes from 'prop-types';
 import * as Rivet from '../util/Rivet';
 
-export interface CheckboxProps {
+const propTypes = {
     /**
      * The checkbox label
      */
-    label: string,
+    label: PropTypes.string.isRequired,
```

Comment:
We can address this in another pass, but I think all labels should be `PropTypes.node`. Someone inevitably is going to want to use markup in their labels.

Reply:
Changed to node